projects/igniteui-angular/src/lib/grids/grid.common.ts

```diff
@@ -125,6 +125,13 @@ export class IgxColumnResizerDirective implements OnInit, OnDestroy {
     }
 }
 
+@Directive({
+    selector: '[igxQuickFilterTemplate]'
+})
+export class IgxQuickFilterTemplateDirective {
```

Comment:
Could you please change the name to `IgxFilterCellTemplateDirective`. It's a bit more meaningful for the users.